docs/accessing_data.md

```diff
@@ -119,13 +120,57 @@ See the [getting started with ActiveData] guide for more details.
 
 ## Direct database access
 
-If there are any use-cases that are not possible via one of the above, we can provide read-only
+If the use-cases above aren't sufficient, we can provide read-only
 access to Treeherder's production MySQL RDS replica. Please [file an infrastructure bug]
 requesting that someone from the Treeherder team [grant access to the read-only replica].
 
 Alternatively if write access is required, we can [create a temporary RDS instance] from
 a production database snapshot.
 
+## Import performance data from upstream
+
+If you're working on a fullstack Perfherder bug and the use-cases above still aren't enough, you should ask for
+read-only access to one of Treeherder's MySQL RDS replicas. Please [file an infrastructure bug]
+requesting that someone from the Treeherder team [grant access to the read-only replica].
```

Comment:
This is paragraph is redundant... maybe mention Perfherder development in the "Direct database access" section above?